### PR TITLE
Travis Build System

### PR DESCRIPTION
**Pull request by bcongdon**
`.travis.yml` for automated TravisCI testing.

The Travis config pulls down the [toplevel](https://github.com/TipoftheHats/donation-tracker-toplevel) repo (for settings + django configuration), and our custom `django-paypal` package. Then, it checks out this repo into the toplevel folder and runs the test suite.

There are some failing tests, but the Travis build system works.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+  - "2.7"
```
Comment (bcongdon) [inline .travis.yml:3]: I assume we're using 2.7?

Comment (Lange) [inline .travis.yml]: Yes, the tracker is 2.7, not 3.x.

**Comment by Lange** [comment]
Looks like the build failed to checkout the desired commit. Is this ready to merge?

**Comment by bcongdon** [comment]
Just fixed that - Travis seems to handle PRs slightly differently than regular builds.

Still trying to see if I can fix the tests -- some of them might be due to the test environment not being configured correctly.

**Comment by Lange** [comment]
kk, no rush 👌

**Pull request merged**